exercises/prime-factors/prime_factors_test.exs

```diff
@@ -62,4 +62,16 @@ defmodule PrimeFactorsTest do
   test "93819012551" do
     assert PrimeFactors.factors_for(93819012551) == [11, 9539, 894119]
   end
+
+  @tag :optional
+  @tag timeout: 120000
+  test "performance optimizations" do
+    %{minute: original_minutes, second: original_seconds} = DateTime.utc_now
+    PrimeFactors.factors_for(10000000055) == [5, 2000000011]
```

Comment:
@devonestes Thanks for the PR! My original thought was just to add this test case. The non optimized solution would still pass the test, but very slow. This will make the users to try to optimized solution. But again, it's optional, so personally I don't feel the need to add the time constrain. Because for different computer cpus, the time may vary.